PublicMapping/districtbuilder
# Add Project Evaluate view for compactness

## Overview

Adds project evaluate mode view for compactness.

- Computes the average compactness and displays it in `ProjectEvaluateSummary`
- Display list of districts and their compactness score in metric detail view
- Display chloropleth map in `Map`
- Add legend for chloropleth map

### Checklist


### Demo

![image](https://user-images.githubusercontent.com/66973361/112523315-0638ae00-8d75-11eb-9ea9-8b14ffb782e9.png)

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

- `./scripts/server`
- Open a project and click evaluate
- Click 'Compactness'
- Expect: Compactness view is rendered as pictured

Closes #607
